src/components/Switzerland/Map.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { geoPath, geoTransform } from 'd3-geo';
+import { scaleLinear } from 'd3-scale';
+import ReactTooltip from 'react-tooltip';
+import { TimeZipCodeDistributionEntry } from '../../services/api-types';
+import { scaleQuantile } from 'd3-scale';
+import styled from 'styled-components';
+
+import bbox from '@turf/bbox';
+import relief from './relief.jpg';
```

Comment:
When I first `relief.jpg` in the code I thought that it's pretty unecessary. I've seen it in the UI now though, and I think it looks really good. It's the only really nice looking thing in our application so far.